homeassistant/components/simplisafe/__init__.py

```diff
@@ -337,66 +304,6 @@ async def async_reload_entry(hass, config_entry):
     await hass.config_entries.async_reload(config_entry.entry_id)
 
 
-class SimpliSafeWebsocket:
-    """Define a SimpliSafe websocket "manager" object."""
-
-    def __init__(self, hass, websocket):
-        """Initialize."""
-        self._hass = hass
-        self._websocket = websocket
-
-    @staticmethod
-    def _on_connect():
-        """Define a handler to fire when the websocket is connected."""
-        LOGGER.info("Connected to websocket")
-
-    @staticmethod
-    def _on_disconnect():
-        """Define a handler to fire when the websocket is disconnected."""
-        LOGGER.info("Disconnected from websocket")
-
-    def _on_event(self, event):
-        """Define a handler to fire when a new SimpliSafe event arrives."""
-        LOGGER.debug("New websocket event: %s", event)
-        async_dispatcher_send(
-            self._hass, TOPIC_UPDATE_WEBSOCKET.format(event.system_id), event
-        )
-
-        if event.event_type not in WEBSOCKET_EVENTS_TO_TRIGGER_HASS_EVENT:
-            return
-
-        if event.sensor_type:
-            sensor_type = event.sensor_type.name
-        else:
-            sensor_type = None
-
-        self._hass.bus.async_fire(
-            EVENT_SIMPLISAFE_EVENT,
```

Comment:
We need to update the docs and remove this event.

Reply:
Is it this PR?

https://github.com/home-assistant/home-assistant.io/pull/17732

Reply:
@MartinHjelmare Correct! 👍🏻